sshd.rb

```diff
@@ -0,0 +1,121 @@
+sshd_section = <<EOF
+<sshd>
+  <config>
+    <AcceptEnv config:type="list">
+      <listentry>LANG LC_CTYPE LC_NUMERIC LC_TIME LC_COLLATE LC_MONETARY LC_MESSAGES</listentry> 
+      <listentry>LC_PAPER LC_NAME LC_ADDRESS LC_TELEPHONE LC_MEASUREMENT</listentry> 
+      <listentry>LC_IDENTIFICATION LC_ALL</listentry> 
+    </AcceptEnv>
+    <AllowTcpForwarding config:type="list">
+      <listentry>yes</listentry> 
+    </AllowTcpForwarding>
+    <Banner config:type="list">
+      <listentry>/etc/issue</listentry> 
+    </Banner>
+    <Compression config:type="list">
+      <listentry>yes</listentry> 
+    </Compression>
+    <MaxAuthTries config:type="list">
+      <listentry>6</listentry> 
+    </MaxAuthTries>
+    <PasswordAuthentication config:type="list">
+      <listentry>yes</listentry> 
+    </PasswordAuthentication>
+    <PermitRootLogin config:type="list">
+      <listentry>no</listentry> 
+    </PermitRootLogin>
+    <PermitUserEnvironment config:type="list">
+      <listentry>yes</listentry> 
+    </PermitUserEnvironment>
+    <PrintMotd config:type="list">
+      <listentry>yes</listentry> 
+    </PrintMotd>
+    <Protocol config:type="list">
+      <listentry>2</listentry> 
+    </Protocol>
+    <PubkeyAuthentication config:type="list">
+      <listentry>yes</listentry> 
+    </PubkeyAuthentication>
+    <RSAAuthentication config:type="list">
+      <listentry>yes</listentry> 
+    </RSAAuthentication>
+    <Subsystem config:type="list">
+      <listentry>sftp /usr/lib64/ssh/sftp-server</listentry> 
+    </Subsystem>
+    <UsePAM config:type="list">
+      <listentry>yes</listentry> 
+    </UsePAM>
+    <X11Forwarding config:type="list">
+      <listentry>yes</listentry> 
+    </X11Forwarding>
+  </config>
+  <status config:type="boolean">true</status> 
+</sshd>
+EOF
+
+require "rexml/document"
+require "augeas"
+SSHD = "/etc/ssh/sshd_config"
+
+# Quick hack, it should be possible to configure REXML to accept (or ignore)
+# the config namespace for attributes
+sshd_section.gsub!("config:", "")
```

Comment:
I'm sure it can be done properly with REXML. I just not wanted to invest more time on it, since this script is not even needed by the customer at the end.